src/Controller/Traits/AdminStyleTrait.php

```diff
@@ -52,5 +52,8 @@ protected function addAdminStyle(ElementInterface $element, int $context = null,
             $data['cls'] .= $adminStyle->getElementCssClass() . ' ';
         }
         $data['qtipCfg'] = $adminStyle->getElementQtipConfig();
+        if (method_exists($adminStyle, 'getElementText')) {
```

Comment:
Would it be better to use an Interface with get- and setElementText and check then against the Interface instead of `method_exist`?

Reply:
I was expecting that :laughing:..It was just there to make the tests pass. If decide to bump pimcore/pimcore requirement to 11.1 then we can kick it.

Reply:
I think then it's ok :) What do you think about adding a short comment, that we can remove this if 11.1 is required?